ULC/Repl.hs

```diff
@@ -13,52 +13,61 @@ This is the top level interface between the user and the interpreter.
 module Repl where
 
 -- ULC Imports.
-import ULC         
+import ULC
   (Term(Var), substitute, vars, reductions, reduce)
-import Parser 
+import Parser
   (apply, term, pLet, pTerm, (+++))
 
 
 -- Tool Imports.
-import System.IO             (hFlush, stdout)
-import qualified Data.Map.Lazy as M 
+import qualified Data.Map.Lazy as M
 
+-- Haskeline imports
+import Control.Monad.Trans
+import Control.Monad.Trans.State.Strict
```

Comment:
Please could you use qualified imports like the tool imports. This is consistent across ulc and is good to keep namespacing under control